pages/common/mlr.md

```diff
@@ -26,3 +26,7 @@
 - Receive JSON and format the output as vertical JSON:
 
 `echo '{"hello":"world", "foo":"bar"}' | mlr --ijson --ojson --jvstack cat`
+
+- Filter lines of compressed CSV file (-S ensures numbers are treated as strings to prevent errors):
```

Comment:
Perhaps this will read better?
```suggestion
- Filter lines of a compressed CSV file (`-S` ensures numbers are treated as strings to prevent errors):
```

Reply:
yes, that's better

Reply:
I would suggest something like: Filter lines of a compressed CSV file, treating numbers as strings

Reply:
Marking as unresolved, since the change hasn't actually been made yet.

Reply:
@vapniks, what do you think of my suggested change ? It gets rid of the parentheses and reads smoother IMO.

Reply:
I just added a new commit for that change.